### ISSUE TITLE
onBackPressed

### ISSUE THREAD

**Issue by Youlektech** [Calsign/APDE#27]
Hi,
since the lase update, onBackPressed is not overridden anymore. I can't find any turnaround.
test code : 

void setup() { size(100,100); }
void draw() {}
void onBackPressed() {}

-> back button still makes the app quit

**Comment by EmmanuelPil** [comment]
Meanwhile you could use this code:

```
void setup() { 
  fullScreen(P2D); 
  rectMode(CENTER) ;
} 

void draw() { 
  background(0) ; 
  rect(mouseX, mouseY, width/8, width/8) ;
} 

void keyPressed() {   
  if (key == CODED && keyCode == android.view.KeyEvent.KEYCODE_BACK) {
     keyCode = 0; 
     println("BACK key pressed") ;
     }
}     

```

**Comment by dzaima** [comment]
@EmmanuelPil That does not work in the latest releases of APDE.

---

There was a bit of talk about this problem in [the Processing Android issues](https://github.com/processing/processing-android/issues/414), but nothing seems to be done so far.

**Comment by dzaima** [comment]
This [has been fixed](https://github.com/processing/processing-android/issues/414) in Processing Android, now APDE just needs to update to that Processing version.

**Comment by Calsign** [comment]
@dzaima APDE v0.5.x (available in preview channel) will get the new version of Android mode when the new version of Android mode is released. The fix is there, but the release hasn't been finalized yet.

**Comment by Calsign** [comment]
This has been fixed in v0.5.0-pre3 on the [preview channel](https://plus.google.com/communities/117703520591262292626). Closing.
